src/components/overlay-play/overlay-play.js

```diff
@@ -30,14 +31,38 @@ const mapStateToProps = state => ({
  */
 class OverlayPlay extends BaseComponent {
   state: Object;
+  adsContainerClickHandlerInitialized: boolean;
 
   /**
    * Creates an instance of OverlayPlay.
    * @param {Object} obj obj
    * @memberof OverlayPlay
    */
   constructor(obj: Object) {
-    super({name: 'OverlayPlay', player: obj.player});
+    super({name: 'OverlayPlay', player: obj.player, config: obj.config});
+  }
+
+  /**
+   * Set up click event handler to ads container in order to control play pause
+   *
+   * @returns {void}
+   * @memberof OverlayPlay
+   */
+  componentDidUpdate() {
+    if (!this.adsContainerClickHandlerInitialized && this.props.metadataLoaded) {
+      let player = document.getElementById(this.config.targetId);
+      let adsContainer = [];
+
+      if (player) {
+        adsContainer = player.querySelectorAll('*[id^="ads-container_"]');
```

Comment:
I think that we should move this actually to owner of this overlay - in the ads plugin itself

Reply:
@OrenMe - If this will come with configuration of if to have it or not, it could be a good idea.

Reply:
@dan-ziv WDYT?